hydra-tui/src/Hydra/TUI/Options.hs

```diff
@@ -55,7 +74,7 @@ parseNodeHost =
     ( long "connect"
         <> short 'c'
         <> help "Hydra-node to connect to in the form of <host>:<port>"
-        <> value (Host "0.0.0.0" 4001)
+        <> value (Host "127.0.0.1" 4001)
```

Comment:
Must: not change the default as using localhost did create problems in the past

Reply:
You can not connect **to** 0.0.0.0 so this default here would not make sense. But you can and we would usually connect **to** localhost.
Could it be that you're referring to another default than this one @ch1bo ? 
Otherwise would you have a meaningful default to suggest here?

Reply:
Are you sure? In the past we had certainly problems when the hydra-node, i.e. the server side, did only bind to `127.0.0.1` and you would not be able to connect to `127.0.0.1`. Maybe it works if server binds to `0.0.0.0` and then a local tui client connects to localhost, but I'm not 100% sure and would test this by hand before changing it.

In short, we don't have a test for exactly this case and I would not change it without reason. Does this make sense?

Reply:
According to [Stephen Kitt](https://unix.stackexchange.com/questions/419880/connecting-to-ip-0-0-0-0-succeeds-how-why) sending packet to `0.0.0.0` could work on Linux as the kernel would replace the destination with the source of the packet, sending it to localhost probably.